tst/SdpApiTest.cpp

```diff
@@ -895,6 +964,74 @@ a=rtpmap:102 H264/90000
     });
 }
 
+TEST_P(SdpApiTest_SdpMatch, populateSingleMediaSection_TestH264Fmtp)
+{
+    const std::string offer = sdpOffer();
+    PCHAR sdp = const_cast<PCHAR>(offer.c_str());
+
+    PRtcPeerConnection pRtcPeerConnection = NULL;
+    PRtcRtpTransceiver transceiver1 = NULL;
+    RtcConfiguration rtcConfiguration;
+    RtcMediaStreamTrack track1;
+    RtcRtpTransceiverInit rtcRtpTransceiverInit;
+    RtcSessionDescriptionInit rtcSessionDescriptionInit;
+
+    MEMSET(&rtcConfiguration, 0x00, SIZEOF(RtcConfiguration));
+    MEMSET(&track1, 0x00, SIZEOF(RtcMediaStreamTrack));
+    MEMSET(&rtcSessionDescriptionInit, 0x00, SIZEOF(RtcSessionDescriptionInit));
+
+    EXPECT_EQ(createPeerConnection(&rtcConfiguration, &pRtcPeerConnection), STATUS_SUCCESS);
+    EXPECT_EQ(addSupportedCodec(pRtcPeerConnection, RTC_CODEC_H264_PROFILE_42E01F_LEVEL_ASYMMETRY_ALLOWED_PACKETIZATION_MODE), STATUS_SUCCESS);
+
+    rtcRtpTransceiverInit.direction = RTC_RTP_TRANSCEIVER_DIRECTION_SENDONLY;
+    track1.kind = MEDIA_STREAM_TRACK_KIND_VIDEO;
+    track1.codec = RTC_CODEC_H264_PROFILE_42E01F_LEVEL_ASYMMETRY_ALLOWED_PACKETIZATION_MODE;
+    STRCPY(track1.streamId, "myKvsVideoStream");
+    STRCPY(track1.trackId, "myVideo");
+    EXPECT_EQ(addTransceiver(pRtcPeerConnection, &track1, &rtcRtpTransceiverInit, &transceiver1), STATUS_SUCCESS);
+
+    STRCPY(rtcSessionDescriptionInit.sdp, (PCHAR) sdp);
+    rtcSessionDescriptionInit.type = SDP_TYPE_OFFER;
+    EXPECT_EQ(setRemoteDescription(pRtcPeerConnection, &rtcSessionDescriptionInit), STATUS_SUCCESS);
+    EXPECT_EQ(createAnswer(pRtcPeerConnection, &rtcSessionDescriptionInit), STATUS_SUCCESS);
+
+    EXPECT_PRED_FORMAT2(testing::IsSubstring, match(),
+                        rtcSessionDescriptionInit.sdp) << "Offer:\n" << sdp << "\nAnswer:\n" << rtcSessionDescriptionInit.sdp;
+    closePeerConnection(pRtcPeerConnection);
+    freePeerConnection(&pRtcPeerConnection);
+}
+
+INSTANTIATE_TEST_CASE_P(SdpApiTest_SdpMatch_Chrome, SdpApiTest_SdpMatch,
+                        ::testing::Values(SdpMatch{
+                                              "../samples/SDP/offers/1v1a1d-chrome-linux.txt",
```

Comment:
knit: I think we should put these sample SDPs under `tst/SDP/offers/` as this belongs to unit tests only.